src/cli.rs

```diff
@@ -1,17 +1,65 @@
 use bfint::interpret;
-use std::io::{self, Read};
+use std::fs;
+use std::io::{self};
 
-fn get_code_from_stdin() -> io::Result<Vec<u8>> {
-    let mut buf: Vec<u8> = vec![];
-    io::stdin().read_to_end(&mut buf).expect("aboba");
-    Ok(buf)
+use clap::{Args, Parser, Subcommand};
+
+#[derive(Parser)]
+#[command(author, version, about, long_about = None)]
+#[command(propagate_version = true)]
+struct Cli {
+    #[command(subcommand)]
+    command: Commands,
+}
+
+#[derive(Subcommand)]
+enum Commands {
+    /// Run brainfuck code (--name file, --source str)
+    Run(Run),
+}
+
+#[derive(Args)]
+struct Run {
```

Comment:
I think in future we can add flags --stdout --stdin, to set output/input data of brainfuck code